llvm/test/Transforms/LoopVectorize/RISCV/reg-usage-bf16.ll

```diff
@@ -1,14 +1,11 @@
 ; REQUIRES: asserts
-; RUN: opt -passes=loop-vectorize -mtriple riscv64 -mattr=+v,+zvfbfmin -prefer-predicate-over-epilogue=scalar-epilogue -debug-only=loop-vectorize,vplan --disable-output -riscv-v-register-bit-width-lmul=1 -S < %s 2>&1 | FileCheck %s
-
-; TODO: -prefer-predicate-over-epilogue=scalar-epilogue was added to allow
```

Comment:
A secondary benefit to always computing register pressure is that we don't need to fall back to scalar epilogues to get the debug output anymore and can use the default EVL tail folding.

EVL tail folding doesn't support fixed-length VFs so I've updated the check lines to be scalable.